sos/policies/distros/redhat.py

```diff
@@ -430,11 +433,26 @@ def upload_sftp(self):
                                                        password=_token)
         raise Exception("Could not retrieve valid or anonymous credentials")
 
+    def check_file_too_big(self, archive):
+        size = os.path.getsize(archive)
+        # Lets check if the size is bigger than the limit.
+        # There's really no need to transform the size to Gb,
+        # so we don't need to call any size converter implemented
+        # in tools.py
+        if len(str(size)) >= 10 and (size >= self._max_size_request):
+            self.ui_log.error(
```

Comment:
IMHO warning is the right verbosity level, as we alter the default (or user-defined) behaviour (hence warn) but no real error happens (rather we prevent one).

Reply:
Ack, you are right. We are failing over to SFTP so a warning is probably the best way to go.

Reply:
Fixed in next version.